src/hooks/useTokenBalances.ts

```diff
@@ -44,11 +45,13 @@ async function fetchBalancesForToken(
   }
 }
 
-async function _getBalances(): Promise<TokenBalanceDetails[]> {
-  // TODO: Remove connect once login is done
-  // await walletApi.connect()
+async function _getBalances(walletInfo: WalletInfo): Promise<TokenBalanceDetails[]> {
+  const { userAddress, networkId } = walletInfo
+  console.log('[useTokenBalances] getBalances for %s in network %s', userAddress, networkId)
+  if (!userAddress || !networkId) {
+    return null
```

Comment:
Now that the func can return null the return type is incorrect. I wonder how ts let it slide